src/main/java/teammates/ui/controller/AdminInstructorAccountAddAction.java

```diff
@@ -102,15 +102,22 @@ protected ActionResult execute() throws EntityDoesNotExistException {
             retryUrl = Url.addParamToUrl(retryUrl, Const.ParamsNames.INSTRUCTOR_EMAIL, data.instructorEmail);
             retryUrl = Url.addParamToUrl(retryUrl, Const.ParamsNames.INSTRUCTOR_INSTITUTION, data.instructorInstitution);
                        
-            String errorMessage = "<a href=" + retryUrl + ">Exception in Importing Data, Retry</a>";
-            statusToUser.add(new StatusMessage(errorMessage, StatusMessageColor.DANGER));
-            String message = "<span class=\"text-danger\">Servlet Action failure in AdminInstructorAccountAddAction" + "<br>";
-            message += e.getClass() + ": " + TeammatesException.toStringWithStackTrace(e) + "<br></span>";
-            errorMessage += "<br>" + message;
+            StringBuilder errorMessage = new StringBuilder(100);
+            errorMessage.append("<a href=")
+                        .append(retryUrl)
+                        .append(">Exception in Importing Data, Retry</a>"); // NOPMD
```

Comment:
PMD complains of a ConsecutiveLiteralAppend here, although I can't see one, so I am assuming it's another false positive